ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -45,7 +42,7 @@ else
 
   # notes: jcallen: all new subnets resides on port groups named: ci-vlan-#### where #### is the vlan id.
   vsphere_portgroup="ci-vlan-${vlanid}"
-
+  portgroup_list+=("${vsphere_portgroup}")
```

Comment:
@WenXinWei can you help sync with upstream for your branch, so that append https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh#L98 together.